content/docs/user-guide/assets/scene-settings/meshes-tab.md

```diff
@@ -94,7 +94,11 @@ You can specify up to five LODs (not including the base mesh) that are numbered
 * Choose the {{< icon browse-edit-select-files.svg >}} **Selection list** button to specify the meshes to include in the LOD.
 
 {{< note >}}
-You can add `_lod0`, `_lod1`, `_lod2`, `_lod3`, `_lod4`, and `_lod5` as suffixes to your mesh names to automatically add a Level of Detail modifier and assign the meshes to the appropriate LOD slots. `_lod0` is the base mesh with the highest resolution geometry, textures, and materials and is assigned to the base mesh group. `_lod1` is assigned to LOD slot **\[0\]**, `_lod2` is assigned to LOD slot **\[1\]**, and so on.
+You can use soft name settings to automatically add a Level of Detail modifier and assign the meshes to the appropriate LOD slots. 
```

Comment:
I would ensure that the term `soft name` is used elsewhere in the documentation ie does the user know what is meant by `soft name`?

Reply:
Rephrased it a little bit here to clarify soft naming conventions.